catalog/spatial/registry/registry-federation-admin-service-impl/src/main/java/org/codice/ddf/registry/federationadmin/service/impl/FederationAdminServiceImpl.java

```diff
@@ -108,227 +66,147 @@ public FederationAdminServiceImpl() {
         this.security = security;
     }
 
-    @Override
-    public void addLocalEntry(Metacard metacard)
-            throws SourceUnavailableException, IngestException {
-        Subject systemSubject = security.getSystemSubject();
-        CreateRequest createRequest = new CreateRequestImpl(metacard);
-        createRequest.getProperties()
-                .put(SecurityConstants.SECURITY_SUBJECT, systemSubject);
-
-        catalogFramework.create(createRequest);
+    public void init() {
```

Comment:
This implementation class was 'emptied' for now since the new implementation is coming in the next couple of days. The current implementation in ddf was incompatible with changes in the interface so instead of trying to make it work with the new interface we just removed everything knowing that it would all be replaced when the new PR came in.